### PR TITLE
Added title case transformation command called 'Transform to Title Case'.

### PR DESCRIPTION
**Pull request by ghost**
This is related to #70858 

Before:
before

After:
after

### CODE REVIEW

**Comment by rebornix** [comment]
@jaqra thanks for your contribution! I'd love to call it *Transform to Title Case* considering that *Capital* might be confusing

FYI
* Microsoft Word calls it *Capitalize Each Word*
* Google Docs calls it *Title Case*
* macOS calls it *Transformation -> Capitalize*
* Sublime Text calls it *Title Case*

**Comment by jaqra** [comment]
@rebornix done

File: src/vs/editor/contrib/linesOperations/linesOperations.ts [hunk 1]
```diff
@@ -947,6 +947,28 @@ export class LowerCaseAction extends AbstractCaseAction {
...
+		super({
+			id: 'editor.action.transformToTitlecase',
+			label: nls.localize('editor.transformToTitlecase', "Transform to Title Case"),
+			alias: 'Transform to Title Case',
+			precondition: EditorContextKeys.writable
+		});
+	}
+
+	protected _modifyText(text: string): string {
+		return text.replace(/\S+/g, function (word: string) {
```
Comment (rebornix) [inline src/vs/editor/contrib/linesOperations/linesOperations.ts]: Should we use `editor.wordSeparators` to split words?

**Review by rebornix — CHANGES_REQUESTED** [review]

File: src/vs/editor/contrib/linesOperations/linesOperations.ts (hunk 1 above)
Comment (jaqra) [reply src/vs/editor/contrib/linesOperations/linesOperations.ts]: @rebornix well, however should we exclude single quote char?

For ex:

Excluded:
- `jaqra's commit` -> `Jaqra's Commit` ✔
- `this is 'single quote char'` -> `This Is 'single Quote Char'`❌

Included:
- `jaqra's commit` -> `Jaqra'S Commit`❌
- `this is 'single quote char'` -> `This Is 'Single Quote Char'`✔

My brain is burning 🧠🔥

Comment (jaqra) [reply src/vs/editor/contrib/linesOperations/linesOperations.ts]: I think we should include (if it exists in word separators)

Comment (jaqra) [reply src/vs/editor/contrib/linesOperations/linesOperations.ts]: Is that change acceptable on **`_modifyText` method's parameter**?

Here is some results:

Capture

**Pull request merged**